packages/core/src/editor/managers/BlockManager.ts

```diff
@@ -171,7 +178,7 @@ export class BlockManager<
    * @param update A partial block which defines how the existing block should be changed.
    */
   public updateBlock(
-    blockToUpdate: BlockIdentifier,
+    blockToUpdate: Location,
```

Comment:
should this take a Location?

Reply:
Same as above, it can be resolved to a block.